_posts/2018-01-25-fair.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: OME FAIR
+categories: community
+---
+
+Recently there have been several publications and substantial discussion about
+the FAIR principles (see for example,
+[Wilkinson et al, 2016](https://www.nature.com/articles/sdata201618) and the
+[Force11 Fair Data Principles](https://www.force11.org/group/fairgroup/fairprinciples)).
+Overall, the goal of the FAIR principles are “to facilitate knowledge
+discovery by assisting humans and machines in their discovery of, access to,
+integration and analysis of …scientific data and their associated algorithms
+and workflows.”[^1]  These principles are extremely powerful but as has been
+repeatedly noted, the routine implementation of FAIR principles is a
+significant challenge.
+ 
+Imaging datasets present a particular challenge for implementing FAIR. The
+datasets are large, multidimensional and complex. Perhaps most importantly it
+is probably unrealistic to suggest that a single metadata standard will handle
+the huge diversity of imaging experiments and datasets. In the best possible
+case, it is likely that there will be families of metadata standards or
+flexible APIs, each tuned and designed for accessing specific types of imaging
+metadata.
+ 
+OME has been working on the image data publication problem for many years.
+Our recent work on the [Image Data Resource (IDR)](https://idr.openmicroscopy.org/) is an example of an added
+value database that integrates imaging data from many biological imaging
+datasets and links gene and drug perturbations with cell phenotypes[^2]. IDR
+focuses on reference image datasets, i.e. those datasets that have high levels
+of biological and molecular annotations and have a strong likelihood of re-use
+by the scientific community.
+ 
+Our work on IDR has been well-received and the resource is growing in size and
+usage.  However, IDR doesn’t address more routine data publication, for
+example the datasets that are not reference images, but are associated with a
```

Comment:
I don't find the ``for example`` clear.
maybe replacing ``for example by ``the datasets in IDR are not reference images...``